basket/news/views.py

```diff
@@ -398,13 +398,6 @@ def common_voice_goals(request):
     if form.is_valid():
         # don't send empty values and use ISO formatted date strings
         data = {k: v for k, v in form.cleaned_data.items() if not (v == '' or v is None)}
-        if 'email' not in data:
-            sentry_client.capture('raven.events.Message', request=request)
-            return HttpResponseJSON({
-                'status': 'error',
-                'errors': 'no valid email address'
-            }, 400)
```

Comment:
I'm probably missing something, but why are we removing this check?

Reply:
This was mostly for debugging. I was trying to get more info on what I thought were calls to the view that were missing emails. This shouldn't have been possible because the email field in the form above this section has email as a required field. When I figured out that the problems were the retries and due to the data dict modification this was no longer necessary and would never be called.

Reply:
Makes sense!